### ISSUE TITLE
Nushell 0.92.0 stderr redirect breaking change

### ISSUE THREAD

**Issue by robbienohra** [atuinsh/atuin#1920]
### What did you expect to happen?

Atuin commands to resolve

### What happened?

Deprecation flag regarding `--redirect-stderr` option.

It looks like this flag was deprecated in [Nushell version 0.92.0](https://www.nushell.sh/blog/2024-04-02-nushell_0_92_0.html) (released today)

### Atuin doctor output

```yaml
atuin:
  version: 18.1.0
  sync: null
shell:
  name: nu
  plugins: []
system:
  os: Darwin
  arch: arm64
  version: '14.4'
  disks:
  - name: Macintosh HD
    filesystem: apfs
  - name: Macintosh HD
    filesystem: apfs
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ellie** [comment]
Hey! Thanks for the report

This was resolved in #1913, and will be available in the next release

**Comment by robbienohra** [comment]
Amazing, you rock, @ellie! Thanks again for such a great tool.
